src/EPR.Accreditation.Common.Data/DataModels/Accreditation.cs

```diff
@@ -39,6 +39,9 @@ public class Accreditation : IdBaseEntity
 
         public DateTime? UpdatedOn { get; set; }
 
+        [Column(TypeName = "decimal(10,3)")]
+        public decimal? AccreditationFee { get; set; }
```

Comment:
This is a monetary fee - therefore the decimal(10,3) is not correct. Use a more sensible one for money

Reply:
Done.